assets/styles/layouts/partials/speaker-card.css

```diff
@@ -55,7 +64,32 @@
 }
 
 .speaker-card__social-link {
-    color: var(--color-secondary-darker);
+    color: var(--color-primary);
     font-size: 1.125rem;
+    padding: 0.25rem;
+    opacity: 0.5;
+    transition: border 0.3s ease;
+    position: relative;
+    z-index: 1;
```

Comment:
This doesn't seem necessary, but results in social links being visible over the header.

<img width="708" alt="Screenshot 2024-08-28 at 14 07 45" src="https://github.com/user-attachments/assets/bddcb4a3-d78b-4f4a-9e88-8fe167d0b58b">

```suggestion
```

Reply:
done